grails-webhooks/grails-app/services/webhooks/WebhookService.groovy

```diff
@@ -183,7 +183,7 @@ class WebhookService {
         SaveWebhookRequest saveWebhookRequest = mapperSaveRequest(hookData)
         boolean shouldUpdate = false
         if(saveWebhookRequest.id) {
-            hook = webhookDataProvider.getWebhook(saveWebhookRequest.id)
+            hook = webhookDataProvider.getWebhookByUuid(saveWebhookRequest.uuid)
```

Comment:
Why is this changing? A lookup by id should be much faster.

Reply:
Because when fetching from both rba and gorm the id could be different, the only thing that it's constant is the uuid